apps/server/src/modules/video-conference/uc/video-conference-end.uc.ts

```diff
@@ -16,6 +16,12 @@ export class VideoConferenceEndUc {
 	) {}
 
 	async end(currentUserId: EntityId, scope: ScopeRef): Promise<VideoConference<BBBBaseResponse>> {
+		/* need to be replace with
+		const [authorizableUser, scopeRessource]: [User, TeamEntity | Course] = await Promise.all([
+			this.authorizationService.getUserWithPermissions(userId),
+			this.videoConferenceService.loadScopeRessources(scopeId, scope),
+		]);
+		*/
```

Comment:
Is this a mandatory change for this refactoring?

Reply:
It is a proposal. 
Mandatory is that you load all ressources that are possible together without await between them. 
Mandatory is that the authorisation service get all ressources that they should authorizied. 

But i expect that the loader you need to implement to solve different scopeRessources must be places at this position.